Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
Not Found: /adminpanel/genQr/
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
Not Found: /adminpanel/genQr/
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\crypt_backend\settings.py changed, reloading.
Watching for file changes with StatReloader
New MongoClient connection
New Database connection
sql_command: SELECT "django_migrations"."id", "django_migrations"."app", "django_migrations"."name", "django_migrations"."applied" FROM "django_migrations"
params: ()
Find query: {'projection': ['id', 'app', 'name', 'applied']}
Result: (1, 'contenttypes', '0001_initial', datetime.datetime(2023, 7, 4, 22, 9, 38, 387000))
Result: (2, 'auth', '0001_initial', datetime.datetime(2023, 7, 4, 22, 9, 51, 228000))
Result: (3, 'admin', '0001_initial', datetime.datetime(2023, 7, 4, 22, 9, 53, 923000))
Result: (4, 'admin', '0002_logentry_remove_auto_add', datetime.datetime(2023, 7, 4, 22, 9, 54, 724000))
Result: (5, 'admin', '0003_logentry_add_action_flag_choices', datetime.datetime(2023, 7, 4, 22, 9, 55, 514000))
Result: (6, 'contenttypes', '0002_remove_content_type_name', datetime.datetime(2023, 7, 4, 22, 30, 33, 826000))
Result: (7, 'auth', '0002_alter_permission_name_max_length', datetime.datetime(2023, 7, 4, 22, 30, 34, 868000))
Result: (8, 'auth', '0003_alter_user_email_max_length', datetime.datetime(2023, 7, 4, 22, 30, 35, 664000))
Result: (9, 'auth', '0004_alter_user_username_opts', datetime.datetime(2023, 7, 4, 22, 30, 36, 445000))
Result: (10, 'auth', '0005_alter_user_last_login_null', datetime.datetime(2023, 7, 4, 22, 30, 37, 240000))
Result: (11, 'auth', '0006_require_contenttypes_0002', datetime.datetime(2023, 7, 4, 22, 30, 38, 20000))
Result: (12, 'auth', '0007_alter_validators_add_error_messages', datetime.datetime(2023, 7, 4, 22, 30, 38, 804000))
Result: (13, 'auth', '0008_alter_user_username_max_length', datetime.datetime(2023, 7, 4, 22, 30, 39, 590000))
Result: (14, 'auth', '0009_alter_user_last_name_max_length', datetime.datetime(2023, 7, 4, 22, 30, 40, 376000))
Result: (15, 'auth', '0010_alter_group_name_max_length', datetime.datetime(2023, 7, 4, 22, 30, 41, 160000))
Result: (16, 'auth', '0011_update_proxy_permissions', datetime.datetime(2023, 7, 4, 22, 30, 41, 962000))
Result: (17, 'auth', '0012_alter_user_first_name_max_length', datetime.datetime(2023, 7, 4, 22, 30, 42, 756000))
Result: (18, 'sessions', '0001_initial', datetime.datetime(2023, 7, 4, 22, 30, 44, 184000))
Result: (19, 'userAuthentication', '0001_initial', datetime.datetime(2023, 7, 6, 18, 58, 56, 670000))
Result: (20, 'userAuthentication', '0002_rename_usermodel_usermodeldb', datetime.datetime(2023, 7, 6, 19, 19, 48, 704000))
Result: (21, 'userAuthentication', '0003_delete_usermodeldb', datetime.datetime(2023, 7, 6, 19, 32, 54, 211000))
Result: (22, 'userProfile', '0001_initial', datetime.datetime(2023, 7, 6, 19, 32, 56, 689000))
Result: (23, 'userAuthentication', '0004_initial', datetime.datetime(2023, 7, 6, 20, 16, 49, 797000))
Result: (24, 'userAuthentication', '0005_referraldb_remove_usermodeldb_address_and_more', datetime.datetime(2023, 7, 6, 21, 5, 55, 418000))
Result: (25, 'userAuthentication', '0006_despoit_mypackages_referraldb_bonus1_and_more', datetime.datetime(2023, 7, 9, 21, 4, 21, 583000))
Result: (26, 'userAuthentication', '0007_rename_despoit_deposit', datetime.datetime(2023, 7, 9, 22, 50, 14, 899000))
Result: (27, 'userAuthentication', '0008_deposit_transectionid', datetime.datetime(2023, 7, 10, 13, 16, 47, 168000))
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\crypt_backend\settings.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
NameError("name 'secret_k' is not defined")
Not Found: /adminpanel/genQr/
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Not Found: /auth/user/signin/
Not Found: /auth/user/signin/
Not Found: /auth/user/signin/
Unauthorized: /user/auth/signin/
Internal Server Error: /user/auth/signin/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py", line 143, in signin_view
    return JsonResponse({'success': 'Login successful','token': token ,'is_staff' : is_staff, 'is2fa' : is2fa},status=200)
                                                                                                        ^^^^^
UnboundLocalError: cannot access local variable 'is2fa' where it is not associated with a value

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py", line 147, in signin_view
    return JsonResponse({"error" : e})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\http\response.py", line 691, in __init__
    data = json.dumps(data, cls=encoder, **json_dumps_params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\__init__.py", line 238, in dumps
    **kw).encode(obj)
          ^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 200, in encode
    chunks = self.iterencode(o, _one_shot=True)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 258, in iterencode
    return _iterencode(o, 0)
           ^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\serializers\json.py", line 106, in default
    return super().default(o)
           ^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 180, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type UnboundLocalError is not JSON serializable
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Unauthorized: /user/auth/signin/
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /user/auth/deposit/
Bad Request: /user/auth/deposit/
Bad Request: /user/auth/deposit/
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /user/auth/deposit/
Bad Request: /user/auth/deposit/
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /user/auth/deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py", line 175, in depositView
    form_data = json.loads(request.POST.get('json_data'))
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\__init__.py", line 339, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not NoneType
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /user/auth/deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py", line 177, in depositView
    quantity = form_data['quantity']
               ~~~~~~~~~^^^^^^^^^^^^
TypeError: 'NoneType' object is not subscriptable
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /user/auth/deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py", line 175, in depositView
    form_data = json.loads(request.POST.get('json_data'))
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\__init__.py", line 339, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not NoneType
Internal Server Error: /user/auth/deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py", line 175, in depositView
    form_data = json.loads(request.POST.get('json_data'))
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\__init__.py", line 339, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not NoneType
Internal Server Error: /user/auth/deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py", line 175, in depositView
    form_data = json.loads(request.POST.get('json_data'))
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\__init__.py", line 339, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not NoneType
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /user/auth/deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py", line 175, in depositView
    form_data = json.loads(request.POST.get('json_data'))
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\__init__.py", line 339, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not NoneType
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /user/auth/deposit/
Bad Request: /user/auth/deposit/
Bad Request: /user/auth/deposit/
Bad Request: /user/auth/deposit/
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /user/auth/deposit/
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /user/auth/deposit/
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /user/auth/deposit/
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /user/auth/deposit/
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\crypt_backend\settings.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /user/auth/deposit/
Bad Request: /user/auth/deposit/
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Bad Request: /user/auth/deposit/
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /user/auth/deposit/
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /user/auth/deposit/
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 92, in get_admin_deposit
    return JsonResponse({
           ^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\http\response.py", line 691, in __init__
    data = json.dumps(data, cls=encoder, **json_dumps_params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\__init__.py", line 238, in dumps
    **kw).encode(obj)
          ^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 200, in encode
    chunks = self.iterencode(o, _one_shot=True)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 258, in iterencode
    return _iterencode(o, 0)
           ^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\serializers\json.py", line 106, in default
    return super().default(o)
           ^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 180, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type FieldFile is not JSON serializable
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 57, in get_admin_deposit
    file_url = request.build_absolute_uri(item.file.url)
                                          ^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\fields\files.py", line 65, in url
    self._require_file()
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\fields\files.py", line 40, in _require_file
    raise ValueError(
ValueError: The 'file' attribute has no file associated with it.
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 58, in get_admin_deposit
    file_url = request.build_absolute_uri(item.file.url)
                                          ^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\fields\files.py", line 65, in url
    self._require_file()
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\fields\files.py", line 40, in _require_file
    raise ValueError(
ValueError: The 'file' attribute has no file associated with it.
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 58, in get_admin_deposit
    file_url = request.build_absolute_uri(item.file.url)
                                          ^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\fields\files.py", line 65, in url
    self._require_file()
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\fields\files.py", line 40, in _require_file
    raise ValueError(
ValueError: The 'file' attribute has no file associated with it.
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 58, in get_admin_deposit
    file_url = request.build_absolute_uri(item.file.url)
                                          ^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\fields\files.py", line 65, in url
    self._require_file()
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\fields\files.py", line 40, in _require_file
    raise ValueError(
ValueError: The 'file' attribute has no file associated with it.
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\crypt_backend\settings.py changed, reloading.
Watching for file changes with StatReloader
Not Found: /media/deposit_files/2021-22.png
Not Found: /media/deposit_files/2021-22_fwzoM9e.png
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\crypt_backend\settings.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\models.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 62, in get_admin_deposit
    file_path = os.path.join(settings.MEDIA_ROOT, file_path)
                                                  ^^^^^^^^^
UnboundLocalError: cannot access local variable 'file_path' where it is not associated with a value
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 62, in get_admin_deposit
    file_path = os.path.join(settings.MEDIA_ROOT, file_path)
                                                  ^^^^^^^^^
UnboundLocalError: cannot access local variable 'file_path' where it is not associated with a value
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 62, in get_admin_deposit
    file_path = os.path.join(settings.MEDIA_ROOT, filepath)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen ntpath>", line 143, in join
  File "<frozen genericpath>", line 152, in _check_arg_types
TypeError: join() argument must be str, bytes, or os.PathLike object, not 'FieldFile'
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 62, in get_admin_deposit
    file_path = os.path.join(settings.MEDIA_ROOT, filepath)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen ntpath>", line 143, in join
  File "<frozen genericpath>", line 152, in _check_arg_types
TypeError: join() argument must be str, bytes, or os.PathLike object, not 'FieldFile'
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 73, in get_admin_deposit
    with open(file_path, 'rb') as file:
         ^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\gdas3\\Desktop\\Project-crypt\\project-replica-backend\\crypt_backend\\media\\'
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 73, in get_admin_deposit
    with open(file_path, 'rb') as file:
         ^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\gdas3\\Desktop\\Project-crypt\\project-replica-backend\\crypt_backend\\media\\'
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 74, in get_admin_deposit
    with open(file_path, 'rb') as file:
         ^^^^^^^^^^^^^^^^^^^^^
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\gdas3\\Desktop\\Project-crypt\\project-replica-backend\\crypt_backend\\media\\deposit_files/'
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 74, in get_admin_deposit
    with open(file_path, 'rb') as file:
         ^^^^^^^^^^^^^^^^^^^^^
PermissionError: [Errno 13] Permission denied: 'C:\\Users\\gdas3\\Desktop\\Project-crypt\\project-replica-backend\\crypt_backend\\media\\deposit_files/'
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\crypt_backend\settings.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\crypt_backend\settings.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\models.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\models.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\models.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\models.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\models.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Not Found: /media/myfiles/6517d991be868907758fcf7f
Not Found: /media/deposit/6517d991be868907758fcf7f
Not Found: /media/myfiles/6517d991be868907758fcf7f
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\crypt_backend\settings.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\crypt_backend\settings.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\crypt_backend\settings.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\crypt_backend\settings.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\crypt_backend\settings.py changed, reloading.
Watching for file changes with StatReloader
Not Found: /media/myfiles/6517d991be868907758fcf7f
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\models.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\models.py changed, reloading.
Watching for file changes with StatReloader
Not Found: /media/myfiles/6517d991be868907758fcf7f
Internal Server Error: /admin/userAuthentication/deposit/1675ca7d-fd50-473a-8f86-57ca8e865a64/change/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\loader_tags.py", line 208, in render
    return template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 177, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1070, in render
    return render_value_in_context(output, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1047, in render_value_in_context
    value = str(value)
            ^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\utils\html.py", line 419, in <lambda>
    klass.__str__ = lambda self: mark_safe(klass_str(self))
                                           ^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\forms\boundfield.py", line 34, in __str__
    return self.as_widget()
           ^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\forms\boundfield.py", line 99, in as_widget
    return widget.render(
           ^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\forms\widgets.py", line 280, in render
    context = self.get_context(name, value, attrs)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\forms\widgets.py", line 495, in get_context
    context = super().get_context(name, value, attrs)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\forms\widgets.py", line 333, in get_context
    context = super().get_context(name, value, attrs)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\forms\widgets.py", line 272, in get_context
    "value": self.format_value(value),
             ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\forms\widgets.py", line 491, in format_value
    if self.is_initial(value):
       ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\forms\widgets.py", line 485, in is_initial
    return bool(value and getattr(value, "url", False))
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\fields\files.py", line 66, in url
    return self.storage.url(self.name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\djongo\storage.py", line 132, in url
    raise ValueError("This file is not accessible via a URL.")
ValueError: This file is not accessible via a URL.
Internal Server Error: /admin/userAuthentication/deposit/fe4ef033-bdbb-414f-b2b0-b66b4fae50b0/change/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\loader_tags.py", line 208, in render
    return template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 177, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1070, in render
    return render_value_in_context(output, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\template\base.py", line 1047, in render_value_in_context
    value = str(value)
            ^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\utils\html.py", line 419, in <lambda>
    klass.__str__ = lambda self: mark_safe(klass_str(self))
                                           ^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\forms\boundfield.py", line 34, in __str__
    return self.as_widget()
           ^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\forms\boundfield.py", line 99, in as_widget
    return widget.render(
           ^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\forms\widgets.py", line 280, in render
    context = self.get_context(name, value, attrs)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\forms\widgets.py", line 495, in get_context
    context = super().get_context(name, value, attrs)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\forms\widgets.py", line 333, in get_context
    context = super().get_context(name, value, attrs)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\forms\widgets.py", line 272, in get_context
    "value": self.format_value(value),
             ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\forms\widgets.py", line 491, in format_value
    if self.is_initial(value):
       ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\forms\widgets.py", line 485, in is_initial
    return bool(value and getattr(value, "url", False))
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\fields\files.py", line 66, in url
    return self.storage.url(self.name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\djongo\storage.py", line 132, in url
    raise ValueError("This file is not accessible via a URL.")
ValueError: This file is not accessible via a URL.
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 62, in get_admin_deposit
    file_data = item.file.read()
                ^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\files\utils.py", line 42, in <lambda>
    read = property(lambda self: self.file.read)
                                 ^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\fields\files.py", line 45, in _get_file
    self._require_file()
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\fields\files.py", line 40, in _require_file
    raise ValueError(
ValueError: The 'file' attribute has no file associated with it.
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 62, in get_admin_deposit
    file_data = item.file.read()
                ^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\files\utils.py", line 42, in <lambda>
    read = property(lambda self: self.file.read)
                                 ^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\fields\files.py", line 45, in _get_file
    self._require_file()
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\fields\files.py", line 40, in _require_file
    raise ValueError(
ValueError: The 'file' attribute has no file associated with it.
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 62, in get_admin_deposit
    file_data = item.file.read()
                ^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\files\utils.py", line 42, in <lambda>
    read = property(lambda self: self.file.read)
                                 ^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\fields\files.py", line 45, in _get_file
    self._require_file()
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\fields\files.py", line 40, in _require_file
    raise ValueError(
ValueError: The 'file' attribute has no file associated with it.
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 62, in get_admin_deposit
    file_data = item.file.read()
                ^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\files\utils.py", line 42, in <lambda>
    read = property(lambda self: self.file.read)
                                 ^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\fields\files.py", line 45, in _get_file
    self._require_file()
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\fields\files.py", line 40, in _require_file
    raise ValueError(
ValueError: The 'file' attribute has no file associated with it.
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\models.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\models.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 66, in get_admin_deposit
    file = grid_fs_storage.open(item.file)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\files\storage.py", line 40, in open
    return self._open(name, mode)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\djongo\storage.py", line 79, in _open
    gridfs, filename = self._get_gridfs(path)
                       ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\djongo\storage.py", line 160, in _get_gridfs
    path, filename = os.path.split(path)
                     ^^^^^^^^^^^^^^^^^^^
  File "<frozen ntpath>", line 211, in split
TypeError: expected str, bytes or os.PathLike object, not FieldFile
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 66, in get_admin_deposit
    file = grid_fs_storage.open(item.file)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\files\storage.py", line 40, in open
    return self._open(name, mode)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\djongo\storage.py", line 79, in _open
    gridfs, filename = self._get_gridfs(path)
                       ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\djongo\storage.py", line 160, in _get_gridfs
    path, filename = os.path.split(path)
                     ^^^^^^^^^^^^^^^^^^^
  File "<frozen ntpath>", line 211, in split
TypeError: expected str, bytes or os.PathLike object, not FieldFile
Bad Request: /user/auth/deposit/
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /user/auth/deposit/
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /user/auth/deposit/
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /user/auth/deposit/
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 67, in get_admin_deposit
    file = grid_fs_storage.open(item.file)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\files\storage.py", line 40, in open
    return self._open(name, mode)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\djongo\storage.py", line 79, in _open
    gridfs, filename = self._get_gridfs(path)
                       ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\djongo\storage.py", line 160, in _get_gridfs
    path, filename = os.path.split(path)
                     ^^^^^^^^^^^^^^^^^^^
  File "<frozen ntpath>", line 211, in split
TypeError: expected str, bytes or os.PathLike object, not FieldFile
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 67, in get_admin_deposit
    file = grid_fs_storage.open(item.file)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\files\storage.py", line 40, in open
    return self._open(name, mode)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\djongo\storage.py", line 79, in _open
    gridfs, filename = self._get_gridfs(path)
                       ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\djongo\storage.py", line 160, in _get_gridfs
    path, filename = os.path.split(path)
                     ^^^^^^^^^^^^^^^^^^^
  File "<frozen ntpath>", line 211, in split
TypeError: expected str, bytes or os.PathLike object, not FieldFile
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\gridfs\__init__.py", line 205, in get_version
    doc = next(cursor)
          ^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\pymongo\cursor.py", line 1246, in next
    raise StopIteration
StopIteration

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 68, in get_admin_deposit
    file = grid_fs_storage.open('rb',file_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\files\storage.py", line 40, in open
    return self._open(name, mode)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\djongo\storage.py", line 81, in _open
    return gridfs.get_last_version(filename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\gridfs\__init__.py", line 227, in get_last_version
    return self.get_version(filename=filename, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\gridfs\__init__.py", line 209, in get_version
    raise NoFile("no version %d for filename %r" % (version, filename))
gridfs.errors.NoFile: no version -1 for filename 'rb'
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\gridfs\__init__.py", line 205, in get_version
    doc = next(cursor)
          ^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\pymongo\cursor.py", line 1246, in next
    raise StopIteration
StopIteration

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 68, in get_admin_deposit
    file = grid_fs_storage.open('rb',file_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\files\storage.py", line 40, in open
    return self._open(name, mode)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\djongo\storage.py", line 81, in _open
    return gridfs.get_last_version(filename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\gridfs\__init__.py", line 227, in get_last_version
    return self.get_version(filename=filename, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\gridfs\__init__.py", line 209, in get_version
    raise NoFile("no version %d for filename %r" % (version, filename))
gridfs.errors.NoFile: no version -1 for filename 'rb'
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 68, in get_admin_deposit
    file = grid_fs_storage.open(file_id,'rb')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\files\storage.py", line 40, in open
    return self._open(name, mode)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\djongo\storage.py", line 79, in _open
    gridfs, filename = self._get_gridfs(path)
                       ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\djongo\storage.py", line 160, in _get_gridfs
    path, filename = os.path.split(path)
                     ^^^^^^^^^^^^^^^^^^^
  File "<frozen ntpath>", line 211, in split
TypeError: expected str, bytes or os.PathLike object, not FieldFile
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 68, in get_admin_deposit
    file = grid_fs_storage.open(file_id,'rb')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\files\storage.py", line 40, in open
    return self._open(name, mode)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\djongo\storage.py", line 79, in _open
    gridfs, filename = self._get_gridfs(path)
                       ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\djongo\storage.py", line 160, in _get_gridfs
    path, filename = os.path.split(path)
                     ^^^^^^^^^^^^^^^^^^^
  File "<frozen ntpath>", line 211, in split
TypeError: expected str, bytes or os.PathLike object, not FieldFile
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /user/auth/deposit/
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /user/auth/deposit/
Bad Request: /user/auth/deposit/
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\models.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /user/auth/deposit/
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /user/auth/get-user-details/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py", line 630, in get_user_all
    username = usernameObj.username
               ^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'username'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py", line 662, in get_user_all
    JsonResponse({'error' : e},status=500)
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\http\response.py", line 691, in __init__
    data = json.dumps(data, cls=encoder, **json_dumps_params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\__init__.py", line 238, in dumps
    **kw).encode(obj)
          ^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 200, in encode
    chunks = self.iterencode(o, _one_shot=True)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 258, in iterencode
    return _iterencode(o, 0)
           ^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\serializers\json.py", line 106, in default
    return super().default(o)
           ^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 180, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type AttributeError is not JSON serializable
Internal Server Error: /user/auth/get-user-details/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py", line 630, in get_user_all
    username = usernameObj.username
               ^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'username'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py", line 662, in get_user_all
    JsonResponse({'error' : e},status=500)
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\http\response.py", line 691, in __init__
    data = json.dumps(data, cls=encoder, **json_dumps_params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\__init__.py", line 238, in dumps
    **kw).encode(obj)
          ^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 200, in encode
    chunks = self.iterencode(o, _one_shot=True)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 258, in iterencode
    return _iterencode(o, 0)
           ^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\serializers\json.py", line 106, in default
    return super().default(o)
           ^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 180, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type AttributeError is not JSON serializable
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 71, in get_admin_deposit
    response['Content-Disposition'] = f'attachment; filename="{os.path.basename(file_id)}"'
                                                               ^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen ntpath>", line 242, in basename
  File "<frozen ntpath>", line 211, in split
TypeError: expected str, bytes or os.PathLike object, not GridOut
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 71, in get_admin_deposit
    response['Content-Disposition'] = f'attachment; filename="{os.path.basename(file_id)}"'
                                                               ^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen ntpath>", line 242, in basename
  File "<frozen ntpath>", line 211, in split
TypeError: expected str, bytes or os.PathLike object, not GridOut
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 110, in get_admin_deposit
    return JsonResponse({
           ^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\http\response.py", line 691, in __init__
    data = json.dumps(data, cls=encoder, **json_dumps_params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\__init__.py", line 238, in dumps
    **kw).encode(obj)
          ^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 200, in encode
    chunks = self.iterencode(o, _one_shot=True)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 258, in iterencode
    return _iterencode(o, 0)
           ^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\serializers\json.py", line 106, in default
    return super().default(o)
           ^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 180, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type HttpResponse is not JSON serializable
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 110, in get_admin_deposit
    return JsonResponse({
           ^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\http\response.py", line 691, in __init__
    data = json.dumps(data, cls=encoder, **json_dumps_params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\__init__.py", line 238, in dumps
    **kw).encode(obj)
          ^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 200, in encode
    chunks = self.iterencode(o, _one_shot=True)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 258, in iterencode
    return _iterencode(o, 0)
           ^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\serializers\json.py", line 106, in default
    return super().default(o)
           ^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 180, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type HttpResponse is not JSON serializable
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 86, in get_admin_deposit
    'file' : file_content
             ^^^^^^^^^^^^
UnboundLocalError: cannot access local variable 'file_content' where it is not associated with a value
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 86, in get_admin_deposit
    'file' : file_content
             ^^^^^^^^^^^^
UnboundLocalError: cannot access local variable 'file_content' where it is not associated with a value
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 87, in get_admin_deposit
    'file' : file_content
             ^^^^^^^^^^^^
UnboundLocalError: cannot access local variable 'file_content' where it is not associated with a value
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 87, in get_admin_deposit
    'file' : file_content
             ^^^^^^^^^^^^
UnboundLocalError: cannot access local variable 'file_content' where it is not associated with a value
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 89, in get_admin_deposit
    'file' : file_content
             ^^^^^^^^^^^^
UnboundLocalError: cannot access local variable 'file_content' where it is not associated with a value
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 89, in get_admin_deposit
    'file' : file_content
             ^^^^^^^^^^^^
UnboundLocalError: cannot access local variable 'file_content' where it is not associated with a value
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 90, in get_admin_deposit
    'file' : file_content
             ^^^^^^^^^^^^
UnboundLocalError: cannot access local variable 'file_content' where it is not associated with a value
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 90, in get_admin_deposit
    'file' : file_content
             ^^^^^^^^^^^^
UnboundLocalError: cannot access local variable 'file_content' where it is not associated with a value
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 73, in get_admin_deposit
    file = grid_fs_storage.open(deposit.file_name, 'rb')
                                ^^^^^^^^^^^^^^^^^
AttributeError: 'QuerySet' object has no attribute 'file_name'
Internal Server Error: /adminpanel/get_admin_deposit/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py", line 73, in get_admin_deposit
    file = grid_fs_storage.open(deposit.file_name, 'rb')
                                ^^^^^^^^^^^^^^^^^
AttributeError: 'QuerySet' object has no attribute 'file_name'
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
Not Found: /media/651827a520fc129135e3753e
Not Found: /favicon.ico
Not Found: /media/deposit/2022-23.png
Not Found: /favicon.ico
Not Found: /media/deposit_files/2022-23.png
Not Found: /media/deposit_files/2022-23.png
Not Found: /deposit_files/2022-23.png
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\models.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /user/auth/set-wallet-address/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py", line 755, in set_wallet_address
    wallet_obj =  Deposit.objects.filter(wallet_address = wallet_address).first()
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\query.py", line 1421, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\query.py", line 1439, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\query.py", line 1446, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\sql\query.py", line 1532, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\sql\query.py", line 1562, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\sql\query.py", line 1407, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg)
                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\sql\query.py", line 1217, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\sql\query.py", line 1709, in names_to_path
    raise FieldError(
django.core.exceptions.FieldError: Cannot resolve keyword 'wallet_address' into field. Choices are: date, earning, file, id, payment_id, quantity, status, transactionId, username
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /user/auth/set-wallet-address/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py", line 755, in set_wallet_address
    wallet_obj =  UserModelDB.objects.filter(wallet_address = wallet_address).first()
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\query.py", line 1421, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\query.py", line 1439, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\query.py", line 1446, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\sql\query.py", line 1532, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\sql\query.py", line 1562, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\sql\query.py", line 1407, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg)
                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\sql\query.py", line 1217, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-crypt\.env\Lib\site-packages\django\db\models\sql\query.py", line 1709, in names_to_path
    raise FieldError(
django.core.exceptions.FieldError: Cannot resolve keyword 'wallet_address' into field. Choices are: base_64_qr, email, id, is2fa, referralLink, registrationDate, secret_key, sponsorName, username, walletAddress
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /user/auth/set-wallet-address/
Internal Server Error: /user/auth/set-wallet-address/
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-crypt\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
Bad Request: /user/auth/get-user-withdraw/
Bad Request: /user/auth/get-user-withdraw/
Bad Request: /user/auth/get-user-withdraw/
Bad Request: /user/auth/get-user-withdraw/
Bad Request: /user/auth/get-user-withdraw/
Watching for file changes with StatReloader
Internal Server Error: /user/auth/get-user-details/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-clone\project-replica-backend\crypt_backend\userAuthentication\views.py", line 576, in get_user_all
    username = usernameObj.username
               ^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'username'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-clone\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-clone\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-clone\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-clone\project-replica-backend\crypt_backend\userAuthentication\views.py", line 608, in get_user_all
    JsonResponse({'error' : e},status=500)
  File "C:\Users\gdas3\Desktop\Project-clone\.env\Lib\site-packages\django\http\response.py", line 691, in __init__
    data = json.dumps(data, cls=encoder, **json_dumps_params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\__init__.py", line 238, in dumps
    **kw).encode(obj)
          ^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 200, in encode
    chunks = self.iterencode(o, _one_shot=True)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 258, in iterencode
    return _iterencode(o, 0)
           ^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-clone\.env\Lib\site-packages\django\core\serializers\json.py", line 106, in default
    return super().default(o)
           ^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 180, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type AttributeError is not JSON serializable
Internal Server Error: /user/auth/get-user-details/
Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-clone\project-replica-backend\crypt_backend\userAuthentication\views.py", line 576, in get_user_all
    username = usernameObj.username
               ^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'username'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\gdas3\Desktop\Project-clone\.env\Lib\site-packages\django\core\handlers\exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-clone\.env\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-clone\.env\Lib\site-packages\django\views\decorators\csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-clone\project-replica-backend\crypt_backend\userAuthentication\views.py", line 608, in get_user_all
    JsonResponse({'error' : e},status=500)
  File "C:\Users\gdas3\Desktop\Project-clone\.env\Lib\site-packages\django\http\response.py", line 691, in __init__
    data = json.dumps(data, cls=encoder, **json_dumps_params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\__init__.py", line 238, in dumps
    **kw).encode(obj)
          ^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 200, in encode
    chunks = self.iterencode(o, _one_shot=True)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 258, in iterencode
    return _iterencode(o, 0)
           ^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\Desktop\Project-clone\.env\Lib\site-packages\django\core\serializers\json.py", line 106, in default
    return super().default(o)
           ^^^^^^^^^^^^^^^^^^
  File "C:\Users\gdas3\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 180, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type AttributeError is not JSON serializable
C:\Users\gdas3\Desktop\Project-clone\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-clone\project-replica-backend\crypt_backend\userAuthentication\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-clone\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-clone\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-clone\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\gdas3\Desktop\Project-clone\project-replica-backend\crypt_backend\adminpanel\views.py changed, reloading.
Watching for file changes with StatReloader
Not Found: /favicon.ico
